liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -64,7 +63,21 @@
     private static final String BINARY_SCANNER_MAVEN_TYPE = "jar";
     private static final String BINARY_SCANNER_MAVEN_VERSION = "[0.0.1,)";
 
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE1 = "A working set of features could not be generated due to conflicts " +
+            "between configured features and the application's API usage: %s. Review and update your server configuration and " +
+            "application to ensure they are not using conflicting features and APIs from different levels of MicroProfile, " +
+            "Java EE, or Jakarta EE. Refer to the following set of suggested features for guidance: %s";
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE2 = "A working set of features could not be generated due to conflicts " +
+            "between configured features: %s. Review and update your server configuration to ensure it is not using conflicting " +
+            "features from different levels of MicroProfile, Java EE, or Jakarta EE. Refer to the following set of " +
+            "suggested features for guidance: %s";
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE3 = "A working set of features could not be generated due to conflicts " +
+            "in the application’s API usage: %s. Review and update your application to ensure it is not using conflicting APIs " +
+            "from different levels of MicroProfile, Java EE, or Jakarta EE.";
```

Comment:
More just an FYI, I could not find an API usage case that returns this error either. Perhaps once we have support for MP 4 and Jakarta EE 9 since we know there are some easy conflicts to trigger there.

Reply:
Thank you.